### DISCUSSION TITLE
GraphRAG

### DISCUSSION THREAD

**Discussion by RazyRo** [reorproject/reor#448] [Ideas]
https://blog.gopenai.com/microsoft-graphrag-and-ollama-code-your-way-to-smarter-question-answering-45a57cc5c38b

https://blog.lancedb.com/graphrag-hierarchical-approach-to-retrieval-augmented-generation/

https://github.com/microsoft/graphrag

**Comment by BradKML** [comment]
What are the benefits of Graph-based RAG over Auto-RAG systems and conventional RAGs?

**Reply by RazyRo** [reply]
Graph-based RAG offers significant advantages over conventional RAG, especially in complex scenarios.   (such as Contextual Precision, Improved Efficiency, Enhanced Accuracy, Advanced Reasoning Capabilities, Flexibility and Adaptability. although complex implementation than traditional methods, requiring an understanding and management of graph structures and Data Quality Dependency)
Auto-RAG systems prioritize efficiency but may sacrifice control and nuanced understanding. (good at Automation Efficiency and Scalability but not in Control Over Retrieval Logic and overlooking human-like reasoning or contextual understanding)
Conventional RAG:
Struggles with complex relationships and nuanced queries. (Limited Context Understanding, Neglecting Relationships and Hallucinations)

**Reply by BradKML** [reply]
What are the most popular libraries (by star count) for each of the different RAG modalities? And are there SOTAs better or equivalent to GraphRAG and AutoRAG?

**Reply by RazyRo** [reply]
I've been learning about AI and following LLM developments, but my professional experience doesn't involve LLM work. Therefore, I may not be able to provide highly nuanced details on your specific questions.
still i have found this: https://www.aisharenet.com/en/2024-github-shidaba/
